File: lua/test/exists_test.lua
-- CustomerioApp SDK exists test

local sdk = require("customerio-app_sdk")

describe("CustomerioAppSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { CustomerioAppSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await CustomerioAppSDK.test()
    equal(null !== testsdk, true)
  })

})
